docs/static/contributing-java-plugin.asciidoc

```diff
@@ -0,0 +1,54 @@
+[[contributing-java-plugin]]
+== Contributing a Java Plugin
+
+beta[]
+
+Now you can write your own Java plugin for use with {ls}.
+We have provided instructions and GitHub examples to give
+you a head start. 
+
+Native support for Java plugins in {ls} consists of several components
+including:
+
+* Extensions to the Java execution engine to support running Java plugins in
+Logstash pipelines
+* APIs for developing Java plugins. 
+The APIs are in the `co.elastic.logstash.api` package. 
```

Comment:
The package for the experimental/6.6 release is `co.elastic.logstash.api.v0`. It is `co.elastic.logstash.api` for the beta/6.7 release.